pytorch.sh

```diff
@@ -1,47 +1,17 @@
-package: pytorch
+package: PyTorch
```

Comment:
do we really need to change the name?

Reply:
This is a completely new package. The name is up to you. But anyway, this was for testing by Christian S, so it doesn't need to be merged yet.

Reply:
ah, ok, I thought it was already there on CVMFS. Fine with me.